### DISCUSSION TITLE
IBM watsonx.ai integration

### DISCUSSION THREAD

**Discussion by bernong** [langfuse/langfuse#4584] [Ideas]
### Describe the feature or potential improvement

Access to integration to IBM watsonx.ai to access Granite models which comes with indemnification (also Apache 2.0 license). Helps with building agents for regulatory industries especially financial services. watsonx also comes with other open-weight/source models - Llama and Mistral

**Comment by marcklingen** [comment]
Thanks for raising this. I hoped that IBM would just implement the openai api schema but it seems like they use a bespoke one.

Have you considered using ibm watsonx via a proxy such as LiteLLM?

**Comment by jannikmaierhoefer** [comment]
Hi all, Langfuse now integrates with the [IBM watsonx Orchestrate ADK](https://developer.watson-orchestrate.ibm.com/): https://langfuse.com/integrations/frameworks/watsonx-orchestrate

Let us know if you have any feedback!
